packages/interbit-ui-tools/src/middleware/actions.js

```diff
@@ -52,15 +57,33 @@ const actionCreators = {
       publicChainAlias,
       privateChainAlias,
       sponsoredChainId,
-      privateChainId
+      privateChainId,
+      dependsOnChainAlias: publicChainAlias
     }
   }),
 
   sponsorChainSaga: ({ chainAlias, publicChainAlias }) => ({
     type: actionTypes.SPONSOR_CHAIN_SAGA,
     payload: {
       chainAlias,
-      publicChainAlias
+      publicChainAlias,
+      dependsOnChainAlias: publicChainAlias
```

Comment:
This is really strange to read out of context. I had to reread the sponsor and load chain tests and switch case in the middleware before realizing this is handled in the same case as load chain.

I think a comment would clarify this. (Did I just ask for a comment?)